static/app/components/onboardingWizard/integrationCard.tsx

```diff
@@ -0,0 +1,191 @@
+import styled from '@emotion/styled';
+import {motion} from 'framer-motion';
+
+import Button from 'sentry/components/button';
+import Card from 'sentry/components/card';
+import Link from 'sentry/components/links/link';
+import {IconChevron, IconClose, IconEllipsis} from 'sentry/icons';
+import {t, tct} from 'sentry/locale';
+import PluginIcon from 'sentry/plugins/components/pluginIcon';
+import space from 'sentry/styles/space';
+import {OnboardingCustomComponentProps} from 'sentry/types';
+import trackAdvancedAnalyticsEvent from 'sentry/utils/analytics/trackAdvancedAnalyticsEvent';
+import testableTransition from 'sentry/utils/testableTransition';
+import {OnboardingIntegrations} from 'sentry/views/onboarding/targetedOnboarding/components/integrationSelect';
+
+import SkipConfirm from './skipConfirm';
+
+const MAX_INTEGRATIONS_COUNT = 3;
+const OnboardingIntegrationSet = new Map(OnboardingIntegrations.map(i => [i.slug, i]));
+
+export default function OnboardingIntegrationCard({
+  organization,
+  task,
+  onboardingState,
+  setOnboardingState,
+}: OnboardingCustomComponentProps) {
+  if (!onboardingState) {
+    return null;
+  }
+
+  const handleSkip = () => {
+    setOnboardingState &&
+      setOnboardingState({
+        ...onboardingState,
+        selectedIntegrations: [],
+      });
+  };
+  // Projects selected during onboarding but not received first event
+  const finishedIntegrations = new Set((task.data && task.data.providers) ?? []);
```

Comment:
Why not write this as 
```suggestion
  const finishedIntegrations = new Set(task.data?.providers ?? []);
```